.travis.yml

```diff
@@ -18,12 +18,23 @@ cache:
     - $HOME/.composer/cache
     - vendor
 
+env:
+  global:
+    - DB_USER=wp
+    - DB_PASSWORD=password
+    - DB_NAME=wp_tests
+
 before_install:
   - phpenv config-rm xdebug.ini || true
   - composer self-update
 
 install:
   - composer install -o --prefer-dist --no-interaction
 
+before_script:
+  - mysql -u root -e "GRANT ALL PRIVILEGES ON ${DB_NAME}.* TO ${DB_USER} IDENTIFIED BY '${DB_PASSWORD}';"
+  - mysql -u root -e "CREATE DATABASE ${DB_NAME};"
```

Comment:
Thinking maybe travis db testing integration should be a blog post or something separate. There are many ways to skin this cat (docker, external dbs, etc) so best to leave up to the user's discretion. We shouldn't even need a db for unit tests, that is for e2e tests.